anda/desktops/lomiri-unity/lomiri-download-manager/lomiri-download-manager.spec

```diff
@@ -1,9 +1,9 @@
 %global forgeurl https://gitlab.com/ubports/development/core/lomiri-download-manager
-%global commit 6e4342ae0ef3710343e2dbf0b8da586853625e2e
+%global commit 682c4928a91da598767e0be2496d9c35af7db035
 %forgemeta
 
 Name:       lomiri-download-manager
-Version:    0.5.2
+Version:    0.1.3
```

Comment:
bump epoch?

Reply:
it didn't build, doesn't need to bump epoch